app/views/apps/acts_or_sr_app/acts_submissions/acts.html.erb

```diff
@@ -13,6 +13,14 @@
       <p>Označte listiny, pre ktoré požadujete vyhotovenie kópie.</p>
       <br />
 
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+            Ak potrebujete listiny, ktoré nie sú zdigitalizované, <%= link_to 'kontaktujte súd', 'https://obcan.justice.sk/infosud/-/infosud/zoznam/sud' %>, ktorý vedie obchodný register, v ktorom je požadovaná spoločnosť zapísaná.
```

Comment:
Myslim, ze sa v issue este spominala nejaka UPVS sluzba na toto, aj tu by sme tam mohli dat.

Reply:
Pisala som v issue @jsuchal, to je podla toho co som pochopila sluzba na ziskanie vypisu z ORSR. Oprav ma ale ak sa mylim

Reply:
@luciajanikova sorry domotal som to. V tej issue su 2 veci:
1. listina != vypis - to je ina sluzba a mali by sme to hned v uvode vysvetlit a dat link na poriadnu sluzbu (rpo alebo to co pisal Brano)
2. prazdny stav - toto je ok. 

Ak toto ma riesit len 2. tak to mozeme mergnut.

Reply:
Ano, toto riesi iba 2. 
1. mozeme tiez dorobit, len potrebujem navrh ako to rozumne sformulovat a kam (ci to pojde do navodu, aj do appky)

Reply:
Za mna takto. 
![Uploading image.png…]()